tests/resource/associations.test.js

```diff
@@ -396,6 +396,56 @@ describe('Resource(associations)', function() {
       });
     });
 
+    it('should set to null the associated object by identifier of object nested', function(done) {
```

Comment:
```it('should allow updating an association to null by replacing its primary key', function(done) { ```  maybe?

Sorry to be a stickler about language, but I think it's important that people be able to read this very clearly in the future